README.rst

```diff
@@ -1,6 +1,12 @@
 ========
 Gotthard
 ========
+Gotthard is a command line tool to simplify connecting to a PostgreSQL instance via a bastion_ host.
+This tool is created in the context of zalando-stups_, however it does not depend on it.
```

Comment:
I'm not sure this claim is valid, given that we are using piu and piu configuration file.